ads/_config.js

```diff
@@ -806,6 +806,10 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  uas: {
+    prefetch: 'https://ads.pubmatic.com/AdServer/js/phoenix.js',
```

Comment:
We provide `renderStart` and `noContent` https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis that can be used to improve user experience. It's highly recommended to support these two APIs. Please let us know if there's any questions.

Reply:
Hello @zhouyx 
Thank you for your suggestions.
This seems very useful, but our current tag code does not have support for onResponse event.
Can we implement renderStart and noContent in next release ?

Reply:
Sure, it's really up to you to make the final decision. But want you to be aware of the existence of these two APIs. 
Please make sure to update the config once you have the APIs implemented. Thanks.